test/integration/test_groundlight.py

```diff
@@ -456,86 +456,88 @@ def test_submit_numpy_image(gl: Groundlight, detector: Detector):
-        color = ImageEnhance.Color(noisy_image)
-        noisy_image = color.enhance(random.uniform(0.75, 1))
-        contrast = ImageEnhance.Contrast(noisy_image)
-        noisy_image = contrast.enhance(random.uniform(0.75, 1))
-        brightness = ImageEnhance.Brightness(noisy_image)
-        noisy_image = brightness.enhance(random.uniform(0.75, 1))
-        img_query = gl.submit_image_query(detector=detector.id, image=noisy_image, wait=0)
-        if label is not None:
-            gl.add_label(img_query, label)
-        return img_query
-
-    dog = Image.open("test/assets/dog.jpeg")
-    cat = Image.open("test/assets/cat.jpeg")
-
-    submit_noisy_image(dog, "YES")
-    submit_noisy_image(dog, "YES")
-    submit_noisy_image(cat, "NO")
-    submit_noisy_image(cat, "NO")
-
-    # wait to give enough time to train
-    wait_period = 30  # seconds
-    num_wait_periods = 4  # 2 minutes total
-    result_confidence = 0.6
-    new_dog_query = None
-    new_cat_query = None
-    for _ in range(num_wait_periods):
-        time.sleep(wait_period)
-        new_dog_query = submit_noisy_image(dog)
-        new_cat_query = submit_noisy_image(cat)
-        new_cat_result_confidence = new_cat_query.result.confidence
-        new_dog_result_confidence = new_dog_query.result.confidence
-
-        if (
-            new_cat_result_confidence and new_cat_result_confidence < result_confidence
-        ) or new_cat_query.result.label == "YES":
-            # If the new query is not confident enough, we'll try again
-            continue
-        elif (
-            new_dog_result_confidence and new_dog_result_confidence < result_confidence
-        ) or new_dog_query.result.label == "NO":
-            # If the new query is not confident enough, we'll try again
-            continue
-        else:
-            assert True
-            return
-
-    assert (
-        False
-    ), f"The detector {detector} quality has not improved after two minutes q.v. {new_dog_query}, {new_cat_query}"
-
-
-def test_ask_method_quality(gl: Groundlight, detector: Detector):
-    # asks for some level of quality on how fast ask_ml is and that we will get a confident result from ask_confident
-    fast_always_yes_iq = gl.ask_ml(detector=detector.id, image="test/assets/dog.jpeg", wait=0)
-    assert iq_is_answered(fast_always_yes_iq)
-    name = f"Test {datetime.utcnow()}"  # Need a unique name
-    query = "Is there a dog?"
-    detector = gl.create_detector(name=name, query=query, confidence_threshold=0.8)
-    fast_iq = gl.ask_ml(detector=detector.id, image="test/assets/dog.jpeg", wait=0)
-    assert iq_is_answered(fast_iq)
-    confident_iq = gl.ask_confident(detector=detector.id, image="test/assets/dog.jpeg", wait=180)
-    assert confident_iq.result.confidence > IQ_IMPROVEMENT_THRESHOLD
+# temporarily disabled until the backend bugfix is deployed
+# @pytest.mark.skipif(MISSING_PIL, reason="Needs pillow")  # type: ignore
```

Comment:
Instead of commenting out you might just use `@pytest.mark.skip(reason="Detector improvement is incredibly flaky")` or something like that?